.travis/install_gradle.sh

```diff
@@ -1,10 +1,8 @@
-#!/bin/bash
-set -e
-set -x
+#!/bin/bash -ex
 
 pushd /tmp
 
-wget http://services.gradle.org/distributions/gradle-2.4-all.zip
+wget $(wget -qO- https://services.gradle.org/versions/current | jq -r .downloadUrl)
```

Comment:
I get your point about not being deterministic anymore if always the latest Gradle release is used. On the other hand, we probably do want to get notified if a new Gradle version breaks something.

My PR is not strictly following a single topic, it was more a "simplify code" / "make code more generic" kind of thing than a "cleanup". If you feel so, I can easily move this commit to a new PR.